Project.cpp

```diff
@@ -0,0 +1,57 @@
+#include <iostream>
+using namespace std;
+
+class User{
+    public:
+    string Username;
+    string Email;
+    User(string username,string email){
+        Username = username;
+        Email = email;
+    }
+    void print(){
+        std::cout<<"Username - "<<Username<<endl;
+        std::cout<<"Email - "<<Email<<endl;
+    }
+    void changeUsername(string newUsername){
```

Comment:
Use const references in function parameters to avoid unnecessary copying of strings. [medium]